public/js/main.js

```diff
@@ -185,9 +185,34 @@ filters.addEventListener("click",(event) => {
   const filterSelected = document.getElementsByClassName("selected")[0];
   filterSelected.removeAttribute("class");
   element.className = "selected";
-  renderData();
+  todoListComputed();
 });
 
+listData.addEventListener("dblclick",(event) => {
+  const element = event.target;
+  const item = element.parentNode.parentNode;
```

Comment:
A variable should start with their scope or object are pointing. This case it's task element.
E.g: `item` -> `taskElement`
- [ ] rename